libraries/native/intrinsics.cpp

```diff
@@ -34,10 +34,10 @@ extern "C" {
    void set_blockchain_parameters_packed( char* data, uint32_t datalen ) {
       return intrinsics::get().call<intrinsics::set_blockchain_parameters_packed>(data, datalen);
    }
-   uint32_t get_parameters_packed( char* id, uint32_t idlen, char* data, uint32_t datalen) {
+   uint32_t get_parameters_packed(const char* id, uint32_t idlen, char* data, uint32_t datalen) {
       return intrinsics::get().call<intrinsics::get_parameters_packed>(id, idlen, data, datalen);
    }
-   void set_parameters_packed( char* data, uint32_t datalen ) {
+   void set_parameters_packed(const char* data, uint32_t datalen ) {
       return intrinsics::get().call<intrinsics::set_parameters_packed>(data, datalen);
```

Comment:
so the prototype set_parameters_packed(const char* data, uint32_t datalen ) means the passed in 'data' will not be set or modified, is this correct?

Reply:
Correct.

Reply:
Correct.